src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String DELIMITER = ",";
+    private static final String NEW_LINE = "\n";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
Please, split logic of getStatistic method to 3 separate methods

- readFile() - returns content
- generateReport() -(you pass contents here)
- writeToFile()-(you pass report and filename here )